source/network/palos-troubleshooting.html.md.erb

```diff
@@ -149,4 +149,44 @@ When this happens then there needs to be a cleaning exercise or a hard reboot as
 Link with further details
 
 - [High Disk Space Usage on / root partition and How To Clear](https://knowledgebase.paloaltonetworks.com/KCSArticleDetail?id=kA10g000000ClaJCAS)
-- [AnalysisMan](https://www.analysisman.com/2020/08/pan-diskspace.html)
\ No newline at end of file
+- [AnalysisMan](https://www.analysisman.com/2020/08/pan-diskspace.html)
+
+## Support Tickets with Palo Alto
+
+### **Pre-req**
+You need to have a Palo Alto Customer Support Portal (SCP) account already setup, if not then you can create one in the [CSP](https://support.paloaltonetworks.com/Support/Index/).
```

Comment:
Can we review this and see who needs to be added? Those in BAU / oncall etc

Reply:
Yeah, can leat that to will, its a manual process, most of us are already there but @willwatters can findout hows wants to or needs to be on it and get them added 👍🏽